servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/Tls13Test.java

```diff
@@ -56,10 +56,12 @@
 
     @RegisterExtension
     static final ExecutionContextExtension SERVER_CTX =
-        ExecutionContextExtension.cached("server-io", "server-executor");
+        ExecutionContextExtension.cached("server-io", "server-executor")
+                .setClassLevel(true);
     @RegisterExtension
     static final ExecutionContextExtension CLIENT_CTX =
-        ExecutionContextExtension.cached("client-io", "client-executor");
+        ExecutionContextExtension.cached("client-io", "client-executor")
+                .setClassLevel(true);
```

Comment:
Great catch! jUnit 4.x were able to recognize if a similar extension was a static field or not, but in jUnit5 it works differently. Thank you!